src/antares/service/local_services/area_local.py

```diff
@@ -314,10 +316,25 @@ def delete_renewable_clusters(self, area: Area, renewable_clusters: List[Renewab
 
     def delete_st_storages(self, area: Area, storages: List[STStorage]) -> None:
         raise NotImplementedError
+    
+    def _read_timeseries(self, ts_file_type: TimeSeriesFileType, study_path: Path, area_id: Optional[str] = None, constraint_id: Optional[str] = None, time_series: Optional[pd.DataFrame] = None,) -> pd.DataFrame:
```

Comment:
remove time_series from this call as you shouldn't have to give TS when reading TS